### PR TITLE
allow to force enabling a producer

### PR DESCRIPTION
**Pull request by rpalcolea**

### CODE REVIEW

**Comment by rpalcolea** [comment]
thoughts on this @akhaku @toolbear @dkoszewnik

File: hollow/src/main/java/com/netflix/hollow/api/producer/enforcer/SingleProducerEnforcer.java [hunk 1]
```diff
@@ -39,4 +39,10 @@
      */
     boolean isPrimary();
 
+
+    /**
+     * Force Maring producer to enable processing cycles.
```
Comment (akhaku) [inline hollow/src/main/java/com/netflix/hollow/api/producer/enforcer/SingleProducerEnforcer.java]: `Maring`?

File: hollow/src/main/java/com/netflix/hollow/api/producer/enforcer/AbstractSingleProducerEnforcer.java [hunk 2]
```diff
@@ -34,6 +34,8 @@
 
     protected abstract boolean _isPrimary();
 
+    protected abstract void _force();
```
Comment (akhaku) [inline hollow/src/main/java/com/netflix/hollow/api/producer/enforcer/AbstractSingleProducerEnforcer.java]: Let's implement this to throw an UnsupportedOperationException by default? This way we won't break existing clients?

**Review by akhaku — COMMENTED** [review]
Generally seems good with the changes

**Comment by rpalcolea** [comment]
Thanks for the feedback, @akhaku. Made the changes

**Review by akhaku — APPROVED** [review]
LGTM, but I'll let @toolbear take a look too

**Pull request merged**